CONTRIBUTING.md

```diff
@@ -2,117 +2,63 @@
-
-- `LGTM` without an explicit approval means that the changes look good, but you haven't pulled down the code, run tests locally and thoroughly reviewed it.
-- `Approval` through the GH UI means that you understand the code, documentation/spec is updated in the right places, you have pulled down and tested the code locally. In addition:
-  - You must also think through anything which ought to be included but is not
-  - You must think through whether any added code could be partially combined (DRYed) with existing code
-  - You must think through any potential security issues or incentive-compatibility flaws introduced by the changes
-  - Naming must be consistent with conventions and the rest of the codebase
-  - Code must live in a reasonable location, considering dependency structures (e.g. not importing testing modules in production code, or including example code modules in production code).
-  - if you approve of the PR, you are responsible for fixing any of the issues mentioned here and more
-- If you sat down with the PR submitter and did a pairing review please note that in the `Approval`, or your PR comments.
-- If you are only making "surface level" reviews, submit any notes as `Comments` without adding a review.
-
-### Updating Documentation
-
-If you open a PR on ibc-go, it is mandatory to update the relevant documentation in /docs.
-
-## Forking
-
-Please note that Go requires code to live under absolute paths, which complicates forking.
-While my fork lives at `https://github.com/colin-axner/ibc-go`,
-the code should never exist at `$GOPATH/src/github.com/colin-axner/ibc-go`.
-Instead, we use `git remote` to add the fork as a new remote for the original repo,
-`$GOPATH/src/github.com/cosmos/ibc-go`, and do all the work there.
-
-For instance, to create a fork and work on a branch of it, I would:
-
-- Create the fork on github, using the fork button.
-- Go to the original repo checked out locally (i.e. `$GOPATH/src/github.com/cosmos/ibc-go`)
-- `git remote add fork git@github.com:colin-axner/ibc-go.git`
-
-Now `fork` refers to my fork and `origin` refers to the ibc-go version.
-So I can `git push -u fork main` to update my fork, and make pull requests to ibc-go from there.
-Of course, replace `colin-axner` with your git handle.
-
-To pull in updates from the origin repo, run
-
-- `git fetch origin`
-- `git rebase origin/main` (or whatever branch you want)
-
-Please don't make Pull Requests from `main`.
+so further discussions can be made. We are following this process so all involved parties are in 
+agreement before any party begins coding the proposed implementation. Please use the [ADR template](./docs/architecture/adr-template.md) 
+to scaffold any new ADR. If you would like to see some examples of how these are written refer 
+to [Cosmos SDK ADRs](https://github.com/cosmos/cosmos-sdk/tree/master/docs/architecture)
```

Comment:
I think using `master` would be a broken link?

```suggestion
to [Cosmos SDK ADRs](https://github.com/cosmos/cosmos-sdk/tree/main/docs/architecture)
```

Reply:
I was wondering why the task wasn't failing, but the master link re-directs to main, we should still change it but I didn't realise it would do that.

Reply:
We should reference our own ADR's now that we have some